test/module/irohad/model/command_validate_execute_test.cpp

```diff
@@ -72,8 +72,9 @@ class CommandValidateExecuteTest : public ::testing::Test {
 
   bool validateAndExecute() {
     auto executor = factory->getCommandExecutor(command);
-    return executor->validate(*command, *wsv_query, creator)
-        and executor->execute(*command, *wsv_query, *wsv_command);
+    return executor->validate(*command, *wsv_query, creator.account_id)
```

Comment:
Again variable for `creator.account_id`
btw, executor can be inlined

Reply:
creator.account_id is more flexible for tests